build-validation/scripts/gradle/02-validate-build-caching-local-in-place.sh

```diff
@@ -166,63 +164,40 @@ print_introduction() {
   IFS='' read -r -d '' text <<EOF
 $(print_introduction_title)
 
-This is the second of several experiments designed to help you optimize your
-team's builds. If you are running this experiment as part of a Gradle Enterprise
-Trial, then the experiments will also help you to build the data necessary to
-determine if Gradle Enterprise is useful to your organization.
-
-This script (and the other experiment scripts) will run some of the
-experiment steps for you, but we'll walk you through each step so that you
-know exactly what we are doing, and why.
-
-In this experiment, we will be checking your build to see how well it takes
-advantage of the local build cache. When the build cache is enabled, Gradle
-saves the output from tasks so that the same output can be reused if the
-task is executed again with the same inputs. This is similar to incremental
-build, except that the cache is used across build runs. So even if you
-perform a clean, cached output will be used if the inputs to a task have not
-changed.
-
-To test out the build cache, we'll run two builds (with build caching
-enabled). Both builds will invoke clean and run the same tasks. We will not
-make any changes between each build run.
-
-If the build is taking advantage of the local build cache, then very few (if
-any) tasks should actually execute on the seond build (all of the task
-output should be used from the local cache).
+In this experiment, you will validate how well a given project leverages
+Gradle’s local build cache. A build is considered fully local-cache enabled if
+all tasks avoid performing any work because:
 
-The Gradle Solutions engineer will then work with you to figure out why some
-(if any) tasks ran on the second build, and how to optimize them to take
-advantage of the build cache.
+  * The tasks' inputs have not changed since their last invocation and
```

Comment:
This wording might suggest that the build cache can only use an output from the last build. This is the case for `up-to-date` checking in the first experiment but the build cache can re-use any output with the same inputs.

Maybe it might be better to say
`the tasks' inputs match the inputs of a previous invocation of the task - in this case the previous invocation in this experiment`